lib/rucio/core/account_limit.py

```diff
@@ -91,36 +92,43 @@ def get_rse_account_usage(rse_id: str, *, session: "Session") -> list["RSEAccoun
 
 
 @read_session
-def get_global_account_limits(account: Optional["InternalAccount"] = None, *, session: "Session") -> dict[str, "RSEResolvedGlobalAccountLimitDict"]:
+def get_global_account_limit(account: Optional["InternalAccount"] = None, rse_expression: Optional[str] = None, *,
+                             session: "Session") -> Union[int, float, dict[str, "RSEResolvedGlobalAccountLimitDict"], None]:
```

Comment:
```suggestion
                             session: "Session") -> Optional[int, float, dict[str, "RSEResolvedGlobalAccountLimitDict"]]:
```

Prefer `Optional` over `Union[..., None]`